apps/system/style/software_button_manager/software_button_manager.css

```diff
@@ -122,5 +110,5 @@
 }
 
 #screen:not(.fullscreen-layout-app) #software-buttons-fullscreen-layout {
-  visibility: hidden;
+  visibility: visible;
```

Comment:
Hmmm, that's strange because this css rule only effects a version of the SHB that we currently never display. Or at least are not supposed to. In any case, this causes the fullscreen-layout SHB to display on the homescreen, which shows a slight drop-shadow. So we need to come up with a different approach for the landscape fix.